### ISSUE TITLE
Hovered features are not surfaced above surrounding features

### ISSUE THREAD

**Issue by mz8i** [nismod/infra-risk-vis#28]
When hovering over a map feature, e.g. a power line, the feature is highlighted but it's ordering in relation to neighbouring features stays the same. As a result, if there are any features that overlap with the highlighted one, part of the feature might not be visible when hovering

Other libraries like Leaflet have an option such as `riseOnHover` - need to investigate how to achieve this mechanism with Deck.GL

**Comment by tomalrussell** [comment]
Is this closed after #35 or still an issue?

**Comment by mz8i** [comment]
Separate issue, as the hover highlight relies on deck.gl's autoHighlight.

See example using power lines:

hovered:
![Screenshot_20220203_184957](https://user-images.githubusercontent.com/36160844/152400081-8ff6ea7b-d76f-4db1-90e5-481019cdf74b.png)

selected:
![Screenshot_20220203_185104](https://user-images.githubusercontent.com/36160844/152400700-46562ceb-4c0d-409b-8c3f-f5dc7db64e82.png)

Whether it's a priority is another topic!

**Comment by mz8i** [comment]
With some more work, the new mechanism we're using for selection highlight could be used for hover highlight, too. This would eliminate the above problem.

But performance might be an issue then, as the selection layer needs to run some computation on all features, whenever the selected feature changes. If that had to happen every time the **hovered** feature changes, we could run into performance issues.
Conversely, the built in autoHighlight mechanism is much simpler to ensure performance
